concepts/websockets.textile

```diff
@@ -0,0 +1,591 @@
+
+As enabling as it has been for the web application developer community in the absence of other suitable tools, long polling is tricky to do properly and is fraught with unexpected complications that must be managed. The same goes for other Comet techniques, which are beyond the scope of this article. All in all, long polling is really just a case of MacGuyvering the tools available in order to do something they weren't really designed to do.
+
+A real solution was needed - something which would empower developers with proper TCP/IP socket-style capabilities in a web environment. Such a solution would need to be built _for the web_, and it would need to address all of the concerns that arise when operating in a web environment.
+
+h3(#enter,-websockets). Enter, WebSockets
+
+Around the middle of 2008, the pain and limitations of using Comet when implementing anything truly robust were being felt particularly keenly by developers "Michael Carter":https://en.wikipedia.org/wiki/Michael_Carter_(entrepreneur) and "Ian Hickson":https://en.wikipedia.org/wiki/Ian_Hickson. Through collaboration "on IRC":https://krijnhoetmer.nl/irc-logs/whatwg/20080618#l-1145 and "W3C mailing lists":https://lists.w3.org/Archives/Public/public-whatwg-archive/2008Jun/0165.html, they hatched a plan to introduce a new standard for modern real-time, bidirectional communication on the web, and thus "the name _"WebSocket"_ was coined":https://lists.w3.org/Archives/Public/public-whatwg-archive/2008Jun/0186.html.
+
+The idea made its way into the W3C HTML draft standard and, shortly after, Michael Carter wrote "an article introducing the Comet community to the WebSockets.":http://cometdaily.com/2008/07/04/html5-websocket/ In 2010, Google Chrome 4 was the first browser to ship full support for WebSockets, with other browser vendors following suit over the course of the next few years. In 2011, "RFC 6455 - _The WebSocket Protocol_":https://tools.ietf.org/html/rfc6455 - was published to the IETF website.
+
+Today, "all major browsers have full support for WebSockets":https://caniuse.com/#feat=websockets, even including Internet Explorer 10 and 11. In addition, browsers on both iOS and Android have supported WebSockets since 2013, which means that, all in all, the modern landscape for WebSocket support is very healthy. Much of the "internet of things" runs on some version of Android as well, so WebSocket support on other types of devices is reasonably pervasive too, as of 2018.
+
+h3(#so-what-exactly-are-websockets,-anyway?). So what exactly are WebSockets, anyway?
```

Comment:
Comma in link

Reply:
The sentence has a comma which appears after the link. Is the comment based on how it looks after rendering to HTML, or just what appears to be correct directly within the PR? If the latter is the case, the comma _should_ appear as text following the link, when rendered.

Reply:
Fixed.